matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matcherPaginator.js

```diff
@@ -1,47 +1,60 @@
 var PhenoTips = (function(PhenoTips) {
   var widgets = PhenoTips.widgets = PhenoTips.widgets || {};
   widgets.MatcherPaginator = Class.create({
-    initialize: function(table, domNode, max) {
+    initialize: function(table, domNodes, max) {
       var _this = this;
       this.table = table;
       this.max = max;
-      this.pagesNode = domNode.down(".xwiki-livetable-pagination-content");
-      var prevPagination = domNode.down(".prevPagination");
-      var nextPagination = domNode.down(".nextPagination");
-      prevPagination && prevPagination.observe("click", function(ev) {
-        _this.gotoPrevPage(ev);
+      this.pagesNodes = [];
+      domNodes.each(function(elem){
+          _this.pagesNodes.push(elem.down(".xwiki-livetable-pagination-content"));
       });
-      nextPagination && nextPagination.observe("click", function(ev) {
-        _this.gotoNextPage(ev);
+
+      this.pagesNodes.each(function(elem){
+          elem.down(".prevPagination") && elem.down(".prevPagination").observe("click", function(ev) {
+              _this.gotoPrevPage(ev);
```

Comment:
I don't think this is ever reached.

Reply:
This can't work because the arrow elements are not children of the `xwiki-livetable-pagination-content`. You's need to go up to `.xwiki-livetable-pagination` first to get to the arrows.